### PR TITLE
chore: replace dashboard, add new example

### PR DESCRIPTION
**Pull request by jwcastillo**
add new dashboard version 
image
image

### CODE REVIEW

**Review by javaducky — APPROVED** [review]
LGTM!

One minor note. Would you mind updating the `README.md` to correctly use `samples/test.js` instead of `example/test.js`. Looks like we missed this on the last PR which refactored the location of the scripts.

**Comment by jwcastillo** [comment]
> LGTM!
> 
> One minor note. Would you mind updating the `README.md` to correctly use `samples/test.js` instead of `example/test.js`. Looks like we missed this on the last PR which refactored the location of the scripts.

yes, im working on new readme, with new examples and test case, and grafana dashboard information

File: grafana/dashboards/dashboard-tests.json [hunk 1]
```diff
@@ -21,12 +21,12 @@
       }
     ]
   },
+  "description": "List of k6 test",
```
Comment (codebien) [inline grafana/dashboards/dashboard-tests.json:24]:
```suggestion
  "description": "List of k6 test runs",
```

File: samples/test2.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  vus: 10,
+  duration: '10s',
+  thresholds: {
+    'http_reqs{expected_response:true}': ['rate>10'],
+  },
+};
+
+export default function () {
+  http.get('https://test.k6.io/');
+}
```
Comment (codebien) [inline samples/test2.js:13]: Why are we adding a second example? Is `test.js` not enough?

**Review by codebien — COMMENTED** [review]
Hi @jwcastillo,
thanks for your contribution :pray: This is awesome :tada: 

I left some comments, in addition I think we should add a quick description about the various available dashboards here: https://github.com/grafana/xk6-output-prometheus-remote/blob/main/grafana/dashboards/README.md.

I'm not sure I'm getting how you are computing the `testid` field, can you clarify, please?

**Pull request merged**